Sources/Player/PlaybackEngine/Internal/Events/AsyncScheduler.swift

```diff
@@ -0,0 +1,23 @@
+protocol AsyncScheduler {
```

Comment:
[suggestion] What do you think about splitting each one in their own file for better discoverability

Reply:
Done!